app/.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Something went wrong in your android studio config. These are already in .idea/ . Remove all the things in app/.idea/